packages/admin/src/common/buttons/split/useSplitButton.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+import { SplitButtonContext, SplitButtonContextOptions } from "./SplitButtonContext";
+
+export function useSplitButton(): SplitButtonContextOptions | undefined {
```

Comment:
wouldn't it make sense to name this hook `useSplitButtonContext` to make more clear what you get?

Reply:
i thought about that, and quickly looked into other hooks we use in comet-admin. There is currently only the `useErrorDialog` Hook which uses a Context directly and this doesn't have the postfix Context either.

Reply:
I wouldn't say to include always Context if a context is returned (as a rule), as in many cases it can be considered as implemendation detail.

But in this case `useSplitButton` I'm not sure what I get here. Alternative: `useSplitButtonApi`.

other alternative: don't export it at all and cosider it as private context between SplitButton and SaveButton

Reply:
I will add the Context postfix, thats absolutly fine for me and makes sense